stdlib/public/core/NormalizedCodeUnitIterator.swift

```diff
@@ -134,7 +134,8 @@ extension UnsafeBufferPointer where Element == UInt8 {
   }
 }
 
-internal struct _NormalizedUTF8CodeUnitIterator: IteratorProtocol {
+//FIXME: This should not be public, this is just a workaround for the tests
```

Comment:
Is this a fix that you're planning on making, or is this something that will remain after we merge with master? If the latter, then I think `public // @testable` may make sense.

Or, if you'd like to keep it internal (my preference), but expose the produced code units as testable, you could add a simple extension to String:

```swift
extension String {
  public // @testable
  var _nfcCodeUnits: [UInt8] { return _slicedGuts.withNFCCodeUnits { Array($0) } }
}
```